src/ripple/app/ledger/Consensus.h

```diff
@@ -74,6 +74,11 @@ class Consensus
     void
     setLastCloseTime (std::uint32_t t) = 0;
 
+    /** Get the network time when the last ledger closed */
+    virtual
+    std::uint32_t
+    getLastCloseTime () const = 0;
```

Comment:
It would be really odd to make getLastCloseTime and setLastCloseTime use inconsistent types. And if the setter and getter are changed, the internal stored type should be changed. This change only adds a getter for an existing variable and changing other functions and the type is a bit out of scope. (Though I agree that a cleanup of this type should be done in this class and other related classes.)